src/tests/JIT/Regression/JitBlue/Runtime_76194/Runtime_76194.cs

```diff
@@ -127,6 +121,7 @@ public CrossVirtualAlloc()
             if (_ptr != null && mprotect(_ptr + PageSize, PageSize, PROT_NONE) != 0)
             {
                 munmap(_ptr, 2 * PageSize);
+                _ptr = null;
             }
```

Comment:
`mmap` does not return null on failure, so the check needs to be for -1

Reply:
`VirtualFree` above should also get this double free fix

Reply:
I don't see what's wrong with VirtualFree